polaris-migrator/src/migrations/react-replace-text-components/steps/replace-display-text.ts

```diff
@@ -57,4 +57,11 @@ export function replaceDisplayText<NodeType = ASTNode>(
       insertJSXAttribute(j, element, 'as', 'p');
     }
   });
+
+  source
+    .find(j.Identifier)
+    .filter((path) => path.node.name === localElementName)
+    .forEach((path) => {
+      path.node.name = 'Text';
+    });
```

Comment:
I wonder if we should always comment this case. How do we know there isn't an associated object configuring the props e.g.
```
test(MyComponent, { ...myComponentProps })
```
Are we just relying on TypeScript bring these (potential) errors to our attention?

Reply:
If so, what if a consumer is not using TypeScript?